### PR TITLE
All BoxBilling references (literals, links, components, installer) replaced in favor of FOSSBilling

### PR DESCRIPTION
**Pull request by Komodroid**
Hi... I just replaced all literal string referring `BoxBilling`, as well as al the links for documentation, community, etc.

⚠️ **Notes:**
1.  The copyright notices were left untouched until a resolution in #86 
2. The directories and parts of code referring to `*/boxbilling/*` paths were left untouched as well so some media won't break
3. Replaced the Facebook page reference in the last step of the install by our Discord server widget (Good idea?)
~~4. Unsure on how to proceed in `bin/gettext_header.php`~~

### CODE REVIEW

File: src/bb-config-sample.php [hunk 1]
```diff
@@ -108,7 +108,7 @@
         'host' => getenv('DB_HOST') ?: '127.0.0.1',
 
         /**
-         * The name of the database for BoxBilling.
+         * The name of the database for FOSSBilling.
          */
         'name' => getenv('DB_NAME') ?: 'boxbilling',
```
Comment (HunterNyan) [inline src/bb-config-sample.php:113]: you can change this fossbilling :)

Comment (Komodroid) [reply src/bb-config-sample.php:113]: Sure 🍡

File: src/bb-config-sample.php [hunk 2]
```diff
@@ -174,7 +174,7 @@
          * @since 4.22.0
          * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
          */
-        'user_agent' => 'Mozilla/5.0 (RedHatEnterpriseLinux; Linux x86_64; BoxBilling; +http://boxbilling.org) Gecko/20100101 Firefox/93.0',
+        'user_agent' => 'Mozilla/5.0 (RedHatEnterpriseLinux; Linux x86_64; FOSSBilling; +http://boxbilling.org) Gecko/20100101 Firefox/93.0',
```
Comment (HunterNyan) [inline src/bb-config-sample.php:177]: sam with the url here, we have the https://fossbilling.org

File: src/bb-config-sample.php (hunk 1 above)
Comment (Komodroid) [reply src/bb-config-sample.php:113]: I mean could it break something?

File: src/bb-library/Box/Mod.php [hunk 3]
```diff
@@ -90,7 +90,7 @@ public function getManifest()
             'name'          => $this->mod,
             'description'   => NULL,
             'homepage_url'  => 'https://github.com/boxbilling/',
```
Comment (HunterNyan) [inline src/bb-library/Box/Mod.php:92]: Chane the URL for the homepage :)

File: src/bb-locale/en_US/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+# en_US translation of FOSSBilling Application
+# Copyright 2011-2021 FOSSBilling <info@boxbilling.org>
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: 4\n"
 "POT-Creation-Date: 2015-02-27 08:07+0000\n"
 "PO-Revision-Date: 2021-08-10 09:00+0300\n"
-"Language-Team: BoxBilling TM <info@boxbilling.org>\n"
+"Language-Team: FOSSBilling TM <info@boxbilling.org>\n"
```
Comment (HunterNyan) [inline src/bb-locale/en_US/LC_MESSAGES/messages.po:9]: Can the email be changed to info@fossbilling.org?

**Review by HunterNyan — CHANGES_REQUESTED** [review]

File: src/bb-config-sample.php (hunk 2 above)
Comment (Komodroid) [reply src/bb-config-sample.php]: Done

File: src/bb-library/Box/Mod.php (hunk 3 above)
Comment (Komodroid) [reply src/bb-library/Box/Mod.php]: Done

File: src/bb-locale/en_US/LC_MESSAGES/messages.po (hunk 4 above)
Comment (Komodroid) [reply src/bb-locale/en_US/LC_MESSAGES/messages.po]: Done

File: src/bb-config-sample.php (hunk 1 above)
Comment (Komodroid) [reply src/bb-config-sample.php]: Don't think so

File: src/bb-locale/en_US/LC_MESSAGES/messages.po (hunk 4 above)
Comment (John-S4) [reply src/bb-locale/en_US/LC_MESSAGES/messages.po]: > Can the email be changed to [info@fossbilling.org](mailto:info@fossbilling.org)?

Who is administering that email address?

**Review by John-S4 — CHANGES_REQUESTED** [review]
Some of these files contained copyright notices that have been changed in this PR. 

My current understanding of the Apache 2 license - should have clarification later on today - is that any file which has an existing copyright notice should keep it and can have a new one added, not change the original unless there is no code retained from the original at all. It is not enough to retain the copyright notice in one copyright or readme file.

For now the original copyright messages in these files should be kept and added to.

**Comment by Komodroid** [comment]
dang it sorry I'll correct them

**Comment by Komodroid** [comment]
> dang it sorry I'll correct them

Done. They bypassed the regex that checked for the copyright notice header 🥲

**Comment by John-S4** [comment]
Can you remove the references to 'FOSSBilling PRO' or 'PRO version of FOSSBilling' - I noticed a couple in there that were changed from BoxBilling PRO references that should already not have been there.

**Comment by Komodroid** [comment]
> Can you remove the references to 'FOSSBilling PRO' or 'PRO version of FOSSBilling' - I noticed a couple in there that were changed from BoxBilling PRO references that should already not have been there.

Sure

**Comment by Komodroid** [comment]
> > Can you remove the references to 'FOSSBilling PRO' or 'PRO version of FOSSBilling' - I noticed a couple in there that were changed from BoxBilling PRO references that should already not have been there.
> 
> Sure

Done

**Review by John-S4 — APPROVED** [review]

**Review by evrifaessa — APPROVED** [review]

**Comment by evrifaessa** [comment]
@HunterNyan, can you mark the requested changes resolved?

**Review by HunterNyan — APPROVED** [review]

**Pull request merged**